tools/build.gradle.kts

```
@@ -17,19 +19,58 @@
 plugins {
     id("application")
     id("com.hedera.block.tools")
+    id("com.gradleup.shadow") version "8.3.5"
 }
 
 description = "Hedera Block Stream Tools"
 
 application {
-    mainModule = "com.hedera.block.tools"
     mainClass = "com.hedera.block.tools.BlockStreamTool"
 }
 
+// Generate Manifest with Main-Class and Implementation-Title
+tasks.withType<Jar> {
```

```suggestion
tasks.withType<Jar>().configureEach {
```

Minor tweak to make Gradel do things more lazily
(https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#eager_apis_to_avoid)